src/services/structures/controllers/getDetailDossierReconventionnement.ts

```diff
@@ -78,84 +77,104 @@ const getDetailStructureWithConseillers =
+  async (idStructure: string, idConseiller: string[]) =>
+    app.service(service.misesEnRelation).Model.aggregate([
+      {
+        $match: {
+          'structure.$id': new ObjectId(idStructure),
+          'conseiller.$id': { $in: idConseiller },
+          $and: [checkAccessMiseEnRelation],
+        },
+      },
+      {
+        $project: {
+          dateRecrutement: 1,
+          statut: 1,
+          dateRupture: 1,
+          'conseillerObj.idPG': 1,
+          'conseillerObj.nom': 1,
+          'conseillerObj.prenom': 1,
+          reconventionnement: 1,
+        },
+      },
+    ]);
+
 const getDetailDossierReconventionnement =
   (app: Application) => async (req: IRequest, res: Response) => {
-    const demarcheSimplifiee = app.get('demarche_simplifiee');
-    const idDossier = req.params.id;
+    const idStructure = req.params.id;
     try {
-      const graphQLClient = new GraphQLClient(demarcheSimplifiee.endpoint, {
-        headers: {
-          authorization: `Bearer ${demarcheSimplifiee.token_api}`,
-          'content-type': 'application/json',
-        },
-      });
-
-      const dossier: any | Error = await graphQLClient
-        .request(queryGetDossierReconventionnement, {
-          dossierNumber: parseInt(idDossier, 10),
-        })
-        .catch(() => {
-          return new Error("Le dossier n'existe pas");
-        });
-      if (dossier instanceof Error) {
-        res.status(404).json({
-          message: dossier.message,
-        });
-        return;
-      }
-      const reconventionnement: IReconventionnementDS = {};
       const checkAccessStructures = await checkAccessReadRequestStructures(
         app,
         req,
       );
       const structure = await getDetailStructureWithConseillers(
         app,
         checkAccessStructures,
-      )(parseInt(dossier.dossier.champs[1]?.integerNumber, 10));
+      )(idStructure as string);
       if (structure.length === 0) {
         res.status(404).json({
           message: "La structure n'existe pas",
         });
         return;
       }
-      reconventionnement.structure = {
-        ...reconventionnement.structure,
-        ...structure[0],
-      };
-      reconventionnement.idDossier = dossier.dossier.id;
-      reconventionnement.statut = dossier.dossier.state;
-      reconventionnement.numeroDossier = dossier.dossier.number;
-      reconventionnement.dateDeCreation =
-        dossier.dossier.datePassageEnConstruction;
-      reconventionnement.nbPostesAttribuees = parseInt(
-        dossier.dossier.champs[4]?.integerNumber,
-        10,
-      );
-      reconventionnement.dateFinProchainContrat =
-        dossier.dossier.champs[5]?.date;
-      const typeDossierReconventionnement = getTypeDossierReconventionnement(
-        reconventionnement.structure.insee.entreprise.forme_juridique,
+
+      const typeDossierDs = getTypeDossierDemarcheSimplifiee(
+        structure[0].insee.entreprise.forme_juridique,
```

Comment:
Il y a 2 structures sans forme juridique
```suggestion
        structure[0]?.insee?.entreprise?.forme_juridique,
```